slave-ruby/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM openshift/jenkins-slave-base-centos7
+
+MAINTAINER Adam Saleh <asaleh@redhat.com>
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-ruby-rhel7-docker" \
+      name="openshift3/jenkins-slave-ruby-rhel7" \
+      version="1" \
+      architecture="x86_64" \
+      release="1" \
```

Comment:
all the labels above this line should be removed (they're only relevant to the rhel dockerfile).